test/fixtures/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = [{
   sysReceivingAddress: 'tsys1qq43tjdd753rct3jj39yvr855gytwf3y8p5kuf9',
   xpubTokens: [{ path: "m/84'/1'/0'/0/0", transfers: 1 }],
   expected: {
-    hex: '82000000000101b8c0ca9606da799a289b8aa943adca0c11a9bf7481237457a6d20587104510b60000000000ffffffff020000000000000000346a320188b5aa80380101000008c1045130465586681b7b2264657363223a226348566962476c6a646d46736457553d227d007f000ce17648170000001600140562b935bea44785c6528948c19e944116e4c48702483045022100aaf0a8cd38c7fa307c7cd002ccf1580079f0627a1bf443d7fe4047e922124ed0022005c62273992964eebb98b4e4f9b34a4708b013f7f91b447f4fa9394a7627503f0121032563fbbbbe4d1e551f12d01e213d14830db0ae15a781b9c9ebf7f2fe5786eb2100000000',
+    hex: '82000000000101b8c0ca9606da799a289b8aa943adca0c11a9bf7481237457a6d20587104510b60000000000ffffffff0200e1f50500000000346a320188b5aa80380101000008c1045130465586681b7b2264657363223a226348566962476c6a646d46736457553d227d007f000c008142170000001600140562b935bea44785c6528948c19e944116e4c4870247304402201c9017f15edcf9e4457ba14f8c0173b2a90d52954c88a1b773fb5586961b701d02200416acd907e01f229e3e33b4235e6c75d75356e8df9de76b34f90fed9fec55790121032563fbbbbe4d1e551f12d01e213d14830db0ae15a781b9c9ebf7f2fe5786eb2100000000',
```

Comment:
this Hex should not have been changed :
https://github.com/syscoin/syscoinjs-lib/blob/30ffc1468783376701a5061c32d2ece353761b52/test/fixtures/index.js#LL23C11-L23C517

The actual value on syscoinjs-lib master is still the right one, if you use this one all tests should pass:

82000000000101b8c0ca9606da799a289b8aa943adca0c11a9bf7481237457a6d20587104510b60000000000ffffffff020000000000000000346a320188b5aa80380101000008c1045130465586681b7b2264657363223a226348566962476c6a646d46736457553d227d007f000ce17648170000001600140562b935bea44785c6528948c19e944116e4c48702483045022100aaf0a8cd38c7fa307c7cd002ccf1580079f0627a1bf443d7fe4047e922124ed0022005c62273992964eebb98b4e4f9b34a4708b013f7f91b447f4fa9394a7627503f0121032563fbbbbe4d1e551f12d01e213d14830db0ae15a781b9c9ebf7f2fe5786eb2100000000